drydock/templates/kustomized/tutor13/base/kustomization.yml

```diff
@@ -6,8 +6,15 @@ resources:
 - k8s/deployments.yml
 - k8s/services.yml
 - k8s/volumes.yml
-{%- if not DRYDOCK_DISABLE_JOBS | default(false) %}
-- k8s/jobs.yml
+{%- if DRYDOCK_INIT_JOBS %}
```

Comment:
How can I enable only the default jobs (lms,cms,mysql)?

Reply:
For now, just disable the plugins & you'll have the default jobs